src/ui/Minimap.tsx

```diff
@@ -75,6 +78,9 @@ export function Minimap({ size = 200 }): JSX.Element {
     gl.clearDepth()
     v.subVectors(chassisBody.current!.getWorldPosition(playerPosition), levelCenter)
     player.current!.position.set(screenPosition.x + (v.x / levelDimensions.x) * size, screenPosition.y - (v.z / levelDimensions.z) * size, 0)
+    chassisBody.current!.getWorldDirection(playerRotation)
+    spriteRotation.set(playerRotation.x, playerRotation.z)
```

Comment:
```suggestion
    spriteRotation.set(playerRotation.x, 0)
```

I think playerRotation.z is not needed

Reply:
It doesn't work anymore once I do not set the z value. Did it work on your side?

Reply:
My mistake it is needed!